Source/WebCore/html/HTMLImageElement.idl

```diff
@@ -34,6 +34,9 @@
     [CEReactions=NotNeeded] attribute [AtomString] DOMString? crossOrigin;
     [CEReactions=NotNeeded, Reflect] attribute DOMString useMap;
     [CEReactions=NotNeeded, Reflect] attribute boolean isMap;
+#if defined(ENABLE_SPATIAL_IMAGE_CONTROLS) && ENABLE_SPATIAL_IMAGE_CONTROLS
+    [CEReactions=NotNeeded, Reflect] attribute boolean controls;
```

Comment:
Can we also add EnabledBySetting=SpatialImageControlsEnabled. Otherwise, this attribute will be exposed on HTMLImageElement whenever we build with this option.